public/javascripts/application.js

```diff
@@ -1,161 +1,142 @@
-function ShowHideContent() {
-  var self = this;
+/* global $ */
+/* global jQuery */
+/* global GOVUK */
```

Comment:
I prefer this because it is closer to what we'd do with a 'proper' module system in the future, wherein we have no globals.